local-env/README.md

```diff
@@ -36,3 +36,18 @@ There's an example in `config/examples/billing-entity.yaml`.
 
 Note that the data is lost when the control-api Pod is shut down.
 When this case happens, delete the `be-####-viewer` cluster role and -binding before recreating the objects.
+
+## Invitations
+
+As described in [Invitation API](https://kb.vshn.ch/appuio-cloud/references/architecture/control-api-invitation.html), you can REDEEM invitations.
+
+For easier development, to get the Redeem Links, you can use Kubectl with a template, like this:
```

Comment:
There should be a mention that those links need a dev instance of the portal started on port 4200.